### PR TITLE
Add Meta-API

### PR DESCRIPTION
**Pull request by mat250**
Added https://www.meta-api.io - online platform to automate APIs integration
@matthewtrask I've changed the description to add reference to OpenAPI specification. Thanks for your feedback :)

### CODE REVIEW

File: _data/tools.yml [hunk 1]
```diff
@@ -1872,3 +1872,12 @@
   v2: true
   v3: true
   v3_1: true
+
+- name: Meta-API
+  category: miscellaneous
+  link: https://www.meta-api.io
+  language: SaaS
+  description: A SaaS platform to integrate APIs using OpenAPI specification files and manipulate data with online code editor and automating configuration, authentication, deployment and monitoring.
```
Comment (philsturgeon) [inline _data/tools.yml:1880]:
```suggestion
  description: A SaaS platform to integrate APIs using OpenAPI documents, and manipulation of data with online code editor, and automating configuration, authentication, deployment and monitoring.
```

**Comment by mat250** [comment]
Thanks @philsturgeon for your review :)

**Comment by philsturgeon** [comment]
Thanks for sending the link over! Wanted to check a few more things out to see what this tool was all about, and it looks VERY interesting. I'll have to have a play. :D

**Comment by mat250** [comment]
> Thanks for sending the link over! Wanted to check a few more things out to see what this tool was all about, and it looks VERY interesting. I'll have to have a play. :D

Sure! Have a look and don't hesitate to reach me if you have questions :)

**Review by philsturgeon — APPROVED** [review]

**Pull request merged**